docs/DafnyRef/UserGuide.md

```diff
@@ -1668,3 +1668,27 @@ periods.
   compiled assembly rather than including `DafnyRuntime.cs` in the build
   process.
 
+
+* `-testContracts:<mode>` - test certain function and method contracts
```

Comment:
"test certain ... contracts" -- which ones? all non-ghost ones? all non-ghost ones that are executable?

Reply:
Ah, I updated the help text with what I think is better wording and forgot to update the RM! What do you think of the wording in [`DafnyOptions.cs`](https://github.com/dafny-lang/dafny/blob/master/Source/DafnyCore/DafnyOptions.cs#L1323-L1333)?